src/controlnet_aux/dwpose/__init__.py

```diff
@@ -1,91 +1,46 @@
-# Openpose
-# Original from CMU https://github.com/CMU-Perceptual-Computing-Lab/openpose
-# 2nd Edited by https://github.com/Hzzone/pytorch-openpose
-# 3rd Edited by ControlNet
-# 4th Edited by ControlNet (added face and correct hands)
-
-import os
-os.environ["KMP_DUPLICATE_LIB_OK"]="TRUE"
-
 import cv2
-import torch
 import numpy as np
+from easy_dwpose import DWposeDetector as Detector
```

Comment:
I made this because you guys have an `image_resolution` parameter that I consider irrelevant, the DWpose Detector class in the `controlnet_aux` is used to support it

Reply:
```python
    def __call__(self, input_image: Image, image_resolution: int = 512, output_type: str = "pil", **kwargs):
        detected_map = self.pose_estimation(input_image, output_type=output_type, draw_pose=draw_pose, **kwargs)

        input_image = np.array(input_image)
        img = resize_image(input_image, image_resolution)
        H, W, C = img.shape

        if output_type == "pil":
            detected_map = detected_map.resize((W, H), Image.BILINEAR)
        else:
            detected_map = cv2.resize(detected_map, (W, H), interpolation=cv2.INTER_LINEAR)

        return detected_map
 ```